test/maxrows.test.js

```diff
@@ -26,7 +22,16 @@ describe('cloudant max rows', function() {
     });
     Thing.belongsTo('foo', {model: Foo});
     Foo.hasMany('things', {foreignKey: 'fooId'});
-    db.automigrate(done);
+    db.automigrate(function cleanUpData(err) {
+      if (err) return done(err);
+      Thing.destroyAll(function removeModelInstances(err) {
+        if (err) return done(err);
+        Foo.destroyAll(function removeModelInstances(err) {
```

Comment:
In this case, true....the cb function name doesn't give much extra info than the api name `destroyAll` :p